tutorials/containers/flux-sched-container.rst

```diff
@@ -0,0 +1,232 @@
+.. _flux-sched-container:
+
+====================================
+Quick Start with Flux in a Container
+====================================
+
+Do you want to submit a job to Flux? Here's a short tutorial on how to do so via a container!
+The reason this works is because a Flux instance can be run anywhere...
+
+    | You can run it on a cluster... 🥜️
+    | You can run it alongside Lustre! 📁️
+    | You can run it on a share... 🤗️
+    | You can run it anywhere! 🏔️
+    | You can run it in a container... 📦️
+    | Using Flux? Total no-brainer! 🧠️
+
+Flux has a regularly updated Docker image on `Docker Hub <https://hub.docker.com/u/fluxrm>`_. 
+You will need to `install docker <https://docs.docker.com/engine/install/>`_ first.
+
+----------------------
+Starting the Container
+----------------------
+
+Run the container as follows:
+
+.. code-block:: bash
+
+  # This says "run an interactive terminal"
+  $ docker run -it fluxrm/flux-sched:latest
+
+At this point, even before starting an instance, we can inspect what is going on with Flux.
+
+
+.. code-block:: shell
+
+  # What is the size of resources we have?
+  $ flux getattr size
+  1
+
+  # Inspect networking
+  $ flux overlay status
```

Comment:
This isn't too interesting for an instance of size=1, so it could be dropped.

Reply:
okay will drop - I'll add this to the docker-compose example since that might be more interesting.